exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,1156 @@
+{
```

Comment:
<p>Maybe a smaller number will be better. With 100 docs in the Docker-DB the NB will freeze for a noticeable time. Not very long, but long enough to make the user feel bored.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Yes, 10 is good.